frontend/src/constants/database.js

```diff
@@ -5,10 +5,17 @@ export const COLLECTION_TRIPS = 'trips';
 export const TRIPS_TITLE = 'title';
 export const TRIPS_DESCRIPTION = 'description';
 export const TRIPS_DESTINATION = 'destination';
-export const TRIPS_COLLABORATORS = 'collaborators';
 export const TRIPS_START_DATE = 'start_date';
 export const TRIPS_END_DATE = 'end_date';
-export const TRIPS_UPDATE_TIMESTAMP = 'trip_update_timestamp';
+export const TRIPS_ACCEPTED_COLLABS = 'accepted_collaborator_uid_arr';
+export const TRIPS_PENDING_COLLABS = 'pending_collaborator_uid_arr';
+export const TRIPS_REJECTED_COLLABS = 'rejected_collaborator_uid_arr';
+export const TRIPS_UPDATE_TIMESTAMP = 'update_timestamp';
+/**
+ * NOTE: The following constant corresponds to the general collaborator field in
+ * {@link_RawTripData} and is not an actual field in a trip document.
```

Comment:
```suggestion
 * {@link RawTripData} and is not an actual field in a trip document.
```

?

Reply:
Oops. This is because in VSCode the space shows up as an underscore (see below for reference). 
![image](https://user-images.githubusercontent.com/39711376/89587855-727b9700-d810-11ea-80f2-e803d12f7e43.png)

Fixed.